invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentStats/DocumentStats.js

```diff
@@ -28,8 +28,15 @@ export default class DocumentStats extends Component {
   }
 
   renderStats() {
-    const { isLoading, error, data } = this.props;
-    let renewalCount = sumBy(data.hits, loan => loan.metadata.extension_count);
+    const { isLoading, error, data, document } = this.props;
+    const pastLoans = data.total;
+    const renewalCount = sumBy(
+      data.hits,
+      loan => loan.metadata.extension_count
+    );
+    let avgLoans = pastLoans / document.metadata.items.total;
```

Comment:
To be honest, I would be tempted to improve the search query and instead of using `document.metadata.items.total` I would probably calculate:

```
itemApi
      .list(
        itemApi
          .query()
          .withStatus(invenioConfig.items.available.status)
          .qs()
      )
```

Reply:
@ntarocco to clarify a bit this point for my better understanding. `document.metadata.items.total` is the result of the query `item_search.search_by_document_pid(document_pid).count()`. If I understood correctly you are suggesting here to re-request the number of items, what would be the difference?